stages/org.osbuild.ignition

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python3
+import json
+import sys
+
+
+STAGE_DESC = "Setup ignition so it will be triggered on first boot."
+STAGE_INFO = """
+Create the file '/boot/ignition.firstboot' that will be used by grub,
+if the necessary ignition support is enabled, to create a kernel
+command line option ('ignition_firstboot') that, if included in the
```

Comment:
s/ignition_firstboot/ignition.firstboot/

Reply:
I was thinking of the grub variable, not the actual kernel command line trigger. Will rephrase.